pkg/tracejob/selector.go

```diff
@@ -0,0 +1,77 @@
+package tracejob
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Selector represents a label query to select the entity to be traced.
+// Most commonly this entity would resolve into a host, pod or container.
+type Selector struct {
```

Comment:
I believe there should be a k8s Go package that already defines a label selector. Do you think that should be used instead?

I don't have the link to the appropriate Go package right now.

Reply:
Thanks Sean, Dale and I did initially think about it but we also want to support more operators than are in the k8s spec so not sure if that would work.